app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -61,6 +63,7 @@ def toggle_status
     else
       @status = @conversation.toggle_status
     end
+    Rails.configuration.dispatcher.dispatch('conversation.status_changed', Time.zone.now, conversation: @conversation)
```

Comment:
I was thinking to get rid of the callbacks because they don't work sometimes and raise race conditions.
Anyway, I am following the above suggestion, for now, if we face any issue in the future we can resolve it.